docs/layers/grid-layer.md

```diff
@@ -62,6 +62,23 @@ Color scale domain, default is set to the range of point counts in each cell.
 Color ranges as an array of colors formatted as `[255, 255, 255]`. Default is
 [colorbrewer](http://colorbrewer2.org/#type=sequential&scheme=YlOrRd&n=6) `6-class YlOrRd`.
 
+##### `getColorValue` (Function, optional)
```

Comment:
Is there a reason not to just call this `getValue`?

The description can explain that this is used to calculate value. 

```
##### `getColorValue` (Function, optional)
```
Note to self: It would be nice to be able to show more clearly the expected return type of the accessors (in this case number)